frontend/lib/src/controllers/transaction_controller.dart

```diff
@@ -7,17 +7,23 @@ List<String> transactionTypes = ['Credit', 'Dept', 'Transfer'];
 
 class TransactionController extends GetxController {
   final RxString _selectedTransactionType = transactionTypes[0].obs;
-
   final Rx<DateTime> _selectedTransactionDate = DateTime.now().obs;
-
   final RxBool _transactionSubmitEnabled = false.obs;
+  RxList<Transaction> transactionList = <Transaction>[].obs;
 
   String get selectedTransactionType => _selectedTransactionType.value;
 
   DateTime get selectedTransactionDate => _selectedTransactionDate.value;
 
   bool get transactionSubmitEnabled => _transactionSubmitEnabled.value;
 
+  Future<List<Transaction>> getTransactions({String accountId = ''}) async {
```

Comment:
organize the position of this function, its in the middle differing code sections